demisto_sdk/commands/validate/old_validate_manager.py

```diff
@@ -747,7 +747,7 @@ def is_skipped_file(self, file_path: str) -> bool:
                 path.name == "CommonServerPython.py"
                 and path.parent.parent.name != "Base"
             )
-            or (path.parent.name == LISTS_DIR and path.name.endswith("_data.json"))
+            or (LISTS_DIR in path.parts and path.name.endswith("_data.json"))
```

Comment:
why `LISTS_DIR in path.parts` and not `path.parent.parent.name == LISTS_DIR`? I think it's always two levels deep.